kdl_parser/src/check_kdl_parser.cpp

```diff
@@ -34,45 +34,48 @@
 
 /* Author: Wim Meeussen */
 
+#include <iostream>
+#include <string>
+
 #include "kdl_parser/kdl_parser.hpp"
 #include <kdl/chainfksolverpos_recursive.hpp>
 #include <kdl/frames_io.hpp>
 #include <urdf/model.h>
-#include <iostream>
 
 using namespace KDL;
-using namespace std;
-using namespace urdf;
 
-void printLink(const SegmentMap::const_iterator& link, const std::string& prefix)
+void printLink(const KDL::SegmentMap::const_iterator & link, const std::string & prefix)
```

Comment:
If qualifying with `KDL::` then should `using namespace KDL;` online 45 be deleted?

Reply:
Good call, done now.